src/lib/states.ts

```diff
@@ -146,7 +161,53 @@ export class InverterStates {
                 state: {id: 'PV2Current', name: 'PV2 current', type: 'number', unit: 'A', role: 'value.current'},
                 register: {reg: 32019, type: ModbusDatatype.int16, length: 1, gain: 100}
             },
-
+            // ####################################################################################################################################
+            // MPPT
+           {
+                interval: UpdateIntervalID.LOW,
+                state: {id: 'mppt1power', name: 'MPPT 1 Power', type: 'number', unit: 'kWh', role: 'value.power.active', desc: 'Total input power of MPPT1'},
```

Comment:
Power should be kW ... or
kWh should be energy :wink:

Reply:
It is changed. Dont know what value will be shown, so further changes will be possible